src/app/zap-templates/templates/app/cluster-objects.zapt

```diff
@@ -206,6 +206,14 @@ public:
     {{zapTypeToEncodableClusterObjectType type}} {{asLowerCamelCase name}}{{> cluster_objects_field_init}};
     {{/zcl_event_fields}}
 
+    {{#if event_is_fabric_scoped}}
+    {{#unless event_is_fabric_index_nullable}}
```

Comment:
This does not look right.  `event_is_fabric_scoped` should be false for events that are not fabric-scoped!

Please do a followup to fix ZAP here and then fix this logic.

Reply:
yep, would follow-up it after fixing zap, thanks

Reply:
https://github.com/project-chip/connectedhomeip/issues/14365